src/sentry/static/sentry/app/styles/overflowEllipsis.jsx

```diff
@@ -1,3 +1,5 @@
+import {css} from 'react-emotion';
```

Comment:
ugh sorry some of this is sloppy. I had a somewhat crazy rebase last week and missed some of this stuff. Thanks for catching it.

i think we should keep it and use it on line 1. that way it can be used as a standalone class if needed.